techniques/html/H90.html

```diff
@@ -2,116 +2,91 @@
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>HTML and XHTML controls that use external labels.</p>
-            <p>This technique relates to <span><a href="https://waic.jp/docs/WCAG21/Understanding/labels-or-instructions">Success Criterion 3.3.2: Labels or Instructions</a> (Sufficient as a way to meet <a href="../general/G131">G131: Providing descriptive labels</a>)</span>.
+            <h2>適用（対象）</h2>
+            <p>外部ラベルを用いている HTML 及び XHTML のコントロール</p>
+            <p>これは、次の達成基準に関連する達成方法である: <span><a href="https://waic.jp/docs/WCAG21/Understanding/labels-or-instructions">達成基準 3.3.2: ラベル又は説明を理解する</a> (<a href="../general/G131">G131: 説明的なラベルを提供する</a>と使用したとき十分)</span>
             </p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to provide a clear indication that a specific form
-               control in a Web application or form is required for successful data submission. A
-               symbol or text indicating that the control is required is programmatically associated
-               with the field by using the <code class="el">label</code> element, or the <code class="el">legend</code> for groups of controls associated via <code class="el">fieldset</code>. If a symbol is used, the user is advised of its meaning before the first use. 
-            </p>
+            <h2>解説</h2>
+            <p>この達成方法の目的は、ウェブアプリケーション又はフォームにおける特定のフォームコントロールが、データ送信の成功のために必須であることを明確に示すことである。コントロールが必須であることを示す記号又はテキストは、<code class="el">label</code> 要素、又は <code class="el">fieldset</code> 要素でグループ化されたコントロールに対する <code class="el">legend</code> 要素を用いて、プログラムが解釈できるようにそのコントロールと関連付ける必要がある。記号が用いられている場合には、事前に利用者にその意味を説明する必要がある。
          </section>
          <section id="examples">
-            <h2>Examples</h2>
+            <h2>事例</h2>
             <section class="example" id="example-1-using-text-to-indicate-required-state">
-               <h3>Example 1: Using text to indicate required state</h3>
-               <p>The text field in the example below has the explicit label of "First name (required):".
-                  The <code class="el">label</code> element's  <code class="att">for</code> attribute matches the <code class="att">id</code> attribute of the <code class="el">input</code> element and the <code class="el">label</code> text indicates that the control is required. 
-               </p><pre xml:space="preserve">
-&lt;label for="firstname"&gt;First name (required):&lt;/label&gt; 
+               <h3>事例 1: テキストを用いて必須項目であることを示す</h3>
+               <p>以下のコード例にあるテキストフィールドには、明示的なラベル「名前 (必須)」がある。<code class="el">label</code> 要素の <code class="att">for</code> 属性値が、<code class="el">input</code> 要素の <code class="att">id</code> 属性値と合致していて、<code class="el">label</code> 要素でマークアップされたテキストによってそのコントロールが必須項目であることを示している。</p>
```

Comment:
コード例にあわせて説明文を変更しておきます。
```suggestion
               <p>以下のコード例にあるテキストフィールドには、明示的なラベル「First name (required):」がある。<code class="el">label</code> 要素の <code class="att">for</code> 属性値が、<code class="el">input</code> 要素の <code class="att">id</code> 属性値と合致していて、<code class="el">label</code> 要素でマークアップされたテキストによってそのコントロールが必須項目であることを示している。</p>
```